content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/print_on_validation.rst

```diff
@@ -0,0 +1,156 @@
+package.
+
+.. seealso::
+   - :ref:`Picking list <inventory/warehouses_storage/barcode_picking>`
+   - :doc:`Tracking label <labels>`
+
+After :ref:`enabling the Delivery Slip setting <inventory/shipping_receiving/print_setup>` in the
+:guilabel:`Hardware` configuration options, clicking :guilabel:`Validate` on the desired operation
+type downloads a PDF of the delivery slip.
+
+The delivery slip shows products, quantities, the delivery order reference number, and the total
+order weight.
+
+.. image:: print_on_validation/delivery-slip.png
+   :align: center
+   :alt: Example delivery slip.
+
+Return slip
+===========
+
+Print a *return slip* to include in a delivery for customer return packages. It identifies the
+return, links to the sales order, and includes item details and customer information. It can also
+include specific return instructions for the customer.
+
+After :ref:`enabling the Return Slip setting <inventory/shipping_receiving/print_setup>` in the
+:guilabel:`Hardware` configuration options, clicking :guilabel:`Validate` on the desired operation
+type downloads a PDF of the return slip.
+
+The return slip displays the company's return address, along with barcodes for both the order and
+the return operation.
+
+.. image:: print_on_validation/return-slip.png
+   :align: center
+   :alt: Example return slip.
+
+Product labels
+==============
+
+Print *product labels* to affix to items in an order, providing essential information such as
+product name, barcode, and price.
+
+After navigating to the intended operation type (:menuselection:`Inventory app --> Configuration -->
+Operations Types`), in the :guilabel:`Hardware` tab, tick the :guilabel:`Product Labels` option.
+
+Doing so makes the :guilabel:`Print label as:` drop-down menu visible, where each product label can
+be printed as:
+
+- :guilabel:`2 x 7 with price`: PDF displaying product name, barcode, and price, fitting two rows
+  and seven columns of product labels per page.
+
+  .. spoiler:: Example 2 x 7
+
+     .. image:: print_on_validation/two-seven.png
+        :align: center
+        :alt: Example 2 x 7 with price.
+
+- :guilabel:`4 x 7 with price`: displays product name, barcode, and price, fitting four rows and
+  seven columns of product labels per page.
+
+  .. spoiler:: Example 4 x 7
+
+     .. image:: print_on_validation/four-seven.png
+        :align: center
+        :alt: Example 4 x 7 with price.
+
+- :guilabel:`4 x 12`: displays product name and barcode. Fits four rows and twelve columns of
+  product labels per page.
+
+  .. spoiler:: Example 4 x 12
+
+     .. image:: print_on_validation/four-twelve.png
+        :align: center
+        :alt: Example 4 x 12.
+
+- :guilabel:`4 x 12 with price`: displays product name, barcode, and price. Fits four rows and
+  twelve columns of product labels per page.
+
+- :guilabel:`ZPL Labels`: prints labels in the Zebra Programming Language (ZPL) containing the
```

Comment:
```suggestion
- :guilabel:`ZPL Labels`: prints labels in the :abbr:`ZPL (Zebra Programming Language)` containing the
```

Reply:
Since this is the first instance of mentioning zebra programming language, I decided to type out the full name (: